frontend/src/queries/nodes/DataTable/DataTableExport.tsx

```diff
@@ -54,7 +54,9 @@ export async function startDownload(
             )
         }
         if (exportContext['columns'].includes('person')) {
-            exportContext['columns'] = exportContext['columns'].map((c: string) => (c === 'person' ? 'person.id' : c))
+            exportContext['columns'] = exportContext['columns'].map((c: string) =>
```

Comment:
Thanks a lot for the changes Derek!
Any reason you name the column 'person.distinct_ids.0' over `person.distinct_id`?

Reply:
@anirudhpillai `distinct_id` does not exist as a property of `person`. The back end is doing some special logic (not 100% clear on what's happening) to query all `distinct_ids` for a user, but csvs are not great at handling lists which is why I pick the first one 

Here's what happens if we try `person.distinct_id`:
![image](https://github.com/user-attachments/assets/f65adca7-8994-4a3b-9639-cdf7b41bfcf1)